core/src/test/java/com/adobe/aem/dot/common/analyzer/AnalyzerTest.java

```diff
@@ -322,11 +322,11 @@ public void testReduceViolationList() throws IOException {
 
     CountedRuleViolation first = (CountedRuleViolation) counted.get(0);
     CountedRuleViolation second = (CountedRuleViolation) counted.get(1);
-    assertEquals("First should have count 3", 3, first.getRuleViolationCount());
-    assertEquals("First should have context2", "context2", first.getContext());
-    assertEquals("Second should have count 1", 1, second.getRuleViolationCount());
-    assertEquals("Second should have context1", "context1", second.getContext());
-    assertEquals("Second should have rule Disp-1", "DOTRules:Disp-1", second.getAnalyzerRule().getId());
+    assertEquals("First should have count 1", 1, first.getRuleViolationCount());
+    assertEquals("First should have context1", "context1", first.getContext());
+    assertEquals("Second should have count 3", 3, second.getRuleViolationCount());
+    assertEquals("Second should have context2", "context2", second.getContext());
+    assertEquals("Second should have rule Disp-2", "DOTRules:Disp-2", second.getAnalyzerRule().getId());
```

Comment:
Is it strange that the results changed here?

Reply:
No, it makes sense the sort order has changed due to some violations being raised from Info to Major, and others from Minor to Major.
```
new TreeSet<>(violations);  // Sort and make unique...
```